apps/pie-docs/src/assets/styles/components/_nav.scss

```diff
@@ -94,3 +94,16 @@ $nav-item-left-margin: calc($nav-category-gap + $nav-category-icon-width);
     border-radius: f.$radius-rounded-b;
   }
 }
+
+.c-nav-inner {
+  position: sticky;
+  top: 0;
+  bottom: 0;
+  // This is to prevent the ios safari address bar from covering part of the nav in mobile
+  max-height: 75vh;
```

Comment:
Think this is absolutely fine to get in for launch - would be good to make a ticket to clean up though, as it does look a bit strange when scrolling without the address bar covering it (so would be good to fix it down the line).

Reply:
cheers - fully agree! I've created a ticket :)